### ISSUE TITLE
Cucumber Tests: Allow conflict-free parallel execution

### ISSUE THREAD

**Issue by dsmf** [eclipse-tractusx/item-relationship-service#573]
**As** developer / tester,
**I want** to be able to execute Cucumber tests in parallel,
**so that** there are no conflicts between executions started by multiple persons or branches (and to be able to enable parallel execution for lower time comsumption).

## Acceptance Criteria

- Parallel execution of tests possible without conflicts
- Solution does not decrease readability of the tests (if we cannot find a solution that fulfills this it might be better to leave the tests as they are)

## Out of Scope

- ...

**Comment by dsmf** [comment]
relates to #518, #519
